### ISSUE TITLE
PEP8 Reformatting

### ISSUE THREAD

**Issue by piyanatk** [UKZN-Astronomy/corrcal#16]
Python codes should be reformatted to PEP8. `pylint` can scan the codes and provide feedback on what to change. @ronniyjoseph, do you know if there is way to automatically reformat to PEP8?

**Comment by ronniyjoseph** [comment]
There's a package autopep8 (pip installable) that promises to do this. Although I've never used it.

Pycharm is and IDE (by Jetbrains and **free** for staff and students at universities!) that can do some amount of PEP8 checking and reformatting. I use Pycharm and usually ignore PEP8 suggestions until I am finished with projects and let it reformat the code. I could see if pycharm has useful feedback. 

However, I think a lot of the PEP8 changes might require a bit more insight in the code.

**Comment by piyanatk** [comment]
I use Pycharm too. I will take a look.

**Comment by piyanatk** [comment]
As we discussed, we will postpone PEP8 reformatting until we are ready to merge the restructure branch to master

**Comment by piyanatk** [comment]
PR submitted https://github.com/UKZN-Astronomy/corrcal/pull/21

**Comment by ronniyjoseph** [comment]
I've reviewed the PR and there are a few things that need to be resolved before we can merge into master.

**Comment by piyanatk** [comment]
Close with merge https://github.com/UKZN-Astronomy/corrcal/pull/21
